tests/console/apache.pm

```diff
@@ -115,7 +119,8 @@ sub run {
         assert_script_run 'curl -v http://localhost:8080/';
 
         # Stop both instances
-        assert_script_run 'killall -TERM httpd2-prefork';
+        # killall does not occur on JeOS
+        assert_script_run('kill -TERM $(ps aux| grep [h]ttpd2-prefork| awk \'{print $2}\')');
```

Comment:
Do we really need square brackets in the grep expression? Especially in case you put single symbol in it

Reply:
```
➜  ~ sleep 1000 &
[1] 24586
➜  ~  ps aux | grep sleep        
root     24586  0.0  0.0   2668   756 pts/9    S    08:14   0:00 sleep 1000
root     24599  0.0  0.0   3540   804 pts/9    S+   08:14   0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn sleep
➜  ~  ps aux | grep "[s]leep"
root     24586  0.0  0.0   2668   756 pts/9    S    08:14   0:00 sleep 1000
```

Reply:
it is just a way how to get rid of the grep line in ps output.

Reply:
Ah, I see. Cool trick =)